server/src/shared/database/seeders/20221110023857-recipes.js

```diff
@@ -1,20 +1,40 @@
 const TABLE_NAME = 'recipes';
 
-const createRecipe = (index, isPublished = false) => {
+const createRecipe = (name, categoryId, isPublished = false, imageurl = null) => {
```

Comment:
please make the order of these the same as it is in the fields function in the model